pkg/artifactory/resource_artifactory_remote_helm_repository.go

```diff
@@ -13,11 +13,31 @@ var helmRemoteSchema = mergeSchema(baseRemoteSchema, map[string]*schema.Schema{
 		Description: "Base URL for the translation of chart source URLs in the index.yaml of virtual repos. " +
 			"Artifactory will only translate URLs matching the index.yamls hostname or URLs starting with this base url.",
 	},
+	"external_dependencies_enabled": {
+		Type:        schema.TypeBool,
+		Default:     false,
+		Optional:    true,
+		ForceNew:    true,
+		Description: "When set, external dependencies are rewritten.",
+	},
+	"external_dependencies_patterns": {
+		Type:     schema.TypeList,
+		Optional: true,
+		ForceNew: true,
+		Elem: &schema.Schema{
+			Type: schema.TypeString,
+		},
+		RequiredWith: []string{"external_dependencies_enabled"},
+		Description: "An Allow List of Ant-style path expressions that specify where external dependencies may be downloaded from. " +
+			"By default, this is set to ** which means that dependencies may be downloaded from any external source.",
```

Comment:
Should we be explicit about the default here with `Default:  "**",`?

Reply:
There is no explicit default in remote docker repo: https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/artifactory/resource_artifactory_remote_docker_repository.go#L35-L45

Reply:
> Should we be explicit about the default here with `Default: "**",`?
I think so.

Reply:
I'm not sure how to add a default for a list, tried `Default:  []string{"**"},`.

> 1 error occurred:
>       * resource artifactory_remote_helm_repository: external_dependencies_patterns: Default is not valid for lists or sets

Reply:
Thanks TF!

[Artifactory schema documentation](https://www.jfrog.com/confluence/display/JFROG/Repository+Configuration+JSON#RepositoryConfigurationJSON-RemoteRepository) has no default value for `externalDependenciesPatterns` so is "**" the implicit default behavior from the API?